snapcraft/tests/test_project_loader.py

```diff
@@ -282,18 +282,19 @@ def test_config_adds_vcs_packages_to_build_packages(self):
                 self.make_snapcraft_yaml(yaml_t.format(s[0]))
                 c = project_loader.Config()
 
-                self.assertTrue(
-                    s[1] in c.parts.build_tools,
-                    '{} not found in {}'.format(s[1],
-                                                c.parts.build_tools))
+                if s[1]:
+                    self.assertTrue(
+                        s[1] in c.parts.build_tools,
+                        '{} not found in {}'.format(
+                            s[1], c.parts.build_tools))
 
     def test_config_adds_vcs_packages_to_build_packages_from_types(self):
         scenarios = [
             ('git', 'git'),
             ('hg', 'mercurial'),
             ('mercurial', 'mercurial'),
             ('bzr', 'bzr'),
-            ('tar', 'tar'),
+            ('tar', ''),
```

Comment:
Is this because tar is on the base image? Can we assume that'll always be the case? What if it's removed? What about other distros?